### ISSUE TITLE
NVDA reading Windows logon password

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#9169]
### Steps to reproduce:

(Reported by an educational institution) 

### Actual behavior:
On several occasions during testing, (on different computers and with restarts between to confirm) when a user is typing their password into the Windows logon screen, NVDA reads the password typed, rather than "star, star, star".

### Expected behavior:

NVDA should always read "star, star, star" instead of characters on password fields.

#### NVDA installed/portable/running from source:

Installed and set to start on the Windows logon screen

#### NVDA version:
2018.4

#### Windows version:

Windows Version: 1803, OS Build 17134.112.

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
@Qchristensen is there another screen reader at the same time starting with log on? I.e. Jaws or narator at the same time?

**Comment by Qchristensen** [comment]
The user advised there is no other AT starting, and NVDA is the only screen reader.

**Comment by Adriani90** [comment]
@Qchristensen could you check on an update for this one? Is the user still having this issue with NVDA 2020.1? I cannot reproduce it on Windows 10 1909 update.

**Comment by Qchristensen** [comment]
My email back to the original correspondent was bounced - presumably they are no longer in that position.  I haven't had any other reports of this and if we can't reproduce it any more, I'm happy to close as WorksForMe and reopen if we get more reports.
